extensions/skins/conversation_v1/player.html

```diff
@@ -30,6 +30,35 @@
                    ng-class="{'conversation-skin-progress-dots-iframed': isIframed, 'conversation-skin-progress-dots-not-iframed': !isIframed}">
     </progress-dots>
 
+    <div ng-if="helpHtml" class="conversation-skin-help-card-container">
+      <div class="conversation-skin-help-card">
```

Comment:
Another thought: the current placement of this card seems problematic. If you have a wide screen, and a relatively small supplement, then the feedback will be appearing in the top-right corner of the screen far away from the supplement.

Instead, I think the HTML for this help card should be inside of the supplement card. It should be 'position: absolute' (and something like top: -12px, right: -12px), and the supplement card should be 'position: relative'. This way, the help card will always appear at the top-right corner of the supplement card.